src/test/java/edu/ucsb/cs156/example/controllers/RecommendationRequestControllerTests.java

```diff
@@ -54,29 +54,54 @@ public void logged_out_users_cannot_get_all() throws Exception {
     @WithMockUser(roles = { "USER" })
     @Test
     public void logged_in_users_can_get_all() throws Exception {
```

Comment:
Is this function the same as the one below?

Reply:
originally I had to add more logic into that function in order for my mutations to get covered, but I realized that I just totally missed a test to put into the file, so I deleted the logic I added and now its the original test